scripts/delete_db_and_setup.sh

```diff
@@ -1,6 +1,10 @@
-rm backend/run/db.sqlite3
-python3 backend/manage.py migrate
-echo "Creating account with username \"admin\". "
-echo "Please enter your desired email and password:"
-python3 backend/manage.py createsuperuser  --username admin
-python3 backend/manage.py loaddata backend/cms_export.json
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash -e
```
(Will exit when a command fails)